Mathlib/Order/Atoms.lean

```diff
@@ -940,6 +1043,25 @@ protected theorem isCoatomic_iff [OrderTop α] [OrderTop β] (f : α ≃o β) :
 #align order_iso.is_coatomic_iff OrderIso.isCoatomic_iff
 
 end OrderIso
+section CompleteLattice
+
+variable [CompleteLattice α]
+
+instance [IsUpperModularLattice α] [IsAtomistic α] : IsStronglyAtomic α where
```

Comment:
Ah now I get it. This to me is the one that shouldn't be an instance

Reply:
In general it would help a lot if you could explain why you resolved a conversation: whether you addressed it or ignored it. I am not reviewing from a computer right now so it is really hard for me to figure this information out on my own. It requires swapping tabs and uncollapsing discussions many times